actors.md

```diff
@@ -450,10 +450,11 @@ func CommitSector(sectorID SectorID, commD, commR, commRStar []byte, proof SealP
 		Fatal("sector already committed!")
 	}
 
-	// make sure the miner has enough collateral to add more storage
-	coll = CollateralForSector(miner.SectorSize)
+  // Power of the miner after adding this sector
+  futurePower = miner.power + miner.SectorSize
+  collateralRequired = collateralForPower(futurePower)
 
-  if coll < vm.MyBalance() - miner.ActiveCollateral {
+  if collateralRequired < vm.MyBalance() {
```

Comment:
Should ActiveCollateral be removed from the spec entirely at this point? I confess I'm [still confused](https://github.com/filecoin-project/specs/issues/60#issuecomment-497585831) as to which collateral it is intended to represent.

It's still in the state, `SubmitPost` and `DePledge`.

Reply:
I don't see any need for active collateral. I agree that we should remove it.

Reply:
yep, it should be removed.